src/components/Todo/TodoItem.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface Props {
+  todo: Todo;
+  onDelete?: () => void;
+  isProcessed: boolean;
+}
+
+export const TodoItem: React.FC<Props> = ({
+  todo,
+  onDelete = () => { },
+  isProcessed,
+}) => {
+  const { title, completed } = todo;
+
+  return (
+    <li className="todo__item">
+      <div
+        className={classNames('todo', {
+          completed,
+        })}
+      >
+        <label className="todo__status-label">
+          <input
+            type="checkbox"
+            className="todo__status"
+          // checked
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)